src/vorta/log.py

```diff
@@ -24,6 +28,7 @@ def init_logger(background=False):
 
     # create handlers
     fh = TimedRotatingFileHandler(LOG_DIR / 'vorta.log', when='d', interval=1, backupCount=5)
```

Comment:
Exactly, and I found it's important to keep it 'vorta.log' instead of just 'vorta' because it is set as name of the current log file, and does not go through the `namer` function to append '.log' to the end of it in the way the backup log files do.